src/pages/Beta/Swap/MyPortfolio/PortfolioChart.tsx

```diff
@@ -22,7 +24,21 @@ const PortfolioChart: React.FC<Props> = () => {
     data.push(d)
   }
 
-  const balance = useGetChainBalance()
+  const { account, chainId } = useActiveWeb3React()
+
+  const { data: balance = 0 } = useQuery(
+    'getChainBalance',
+    async () => {
+      if (account && chainId) {
+        return await getChainBalance(account, chainId)
+      }
+      return 0
+    },
+    {
+      // Refetch the data every 10 seconds
+      refetchInterval: 10000
+    }
+  )
```

Comment:
I think this entire logic should be moved to `src/state/portifolio/fetch.ts`, something like
```js
const useGetChainBalance = () => {
  const { account, chainId } = useActiveWeb3React()

  return useQuery(
    'getChainBalance',
    async () => {
      if (account && chainId) {
        return await getChainBalance(account, chainId)
      }
      return 0
    },
    {
      // Refetch the data every 10 seconds
      refetchInterval: 10000
    }
  )
}
```

then we just need to use 
```js
const { data } = useGetChainBalance()
```

here

we should do same for every useQuery in this PR
what do you think ?